payjoin/Cargo.toml

```diff
@@ -18,7 +18,7 @@ exclude = ["tests"]
 send = []
 receive = ["rand"]
 base64 = ["bitcoin/base64"]
-v2 = ["bitcoin/rand-std", "chacha20poly1305", "ohttp", "bhttp", "serde"]
+v2 = ["bitcoin/rand-std", "bitcoin/serde", "chacha20poly1305", "ohttp", "bhttp", "serde"]
```

Comment:
umm how do we feel about exporting two serdes here?

Reply:
neither of these features export serde.

`bitcoin/serde` implements serde-based serialization and deserialization on bitcoin data structures

`serde` allows us to implement serde-based serialization and deserialization on our own types